server/ai_session.go

```diff
@@ -215,9 +217,33 @@ func NewAISessionSelector(ctx context.Context, cap core.Capability, modelID stri
 		return nil, err
 	}
 
+	// Periodically refresh sessions for Live Video to Video in order to minimize the necessity of refreshing sessions
+	// when the AI process is started
+	if cap == core.Capability_LiveVideoToVideo {
+		startPeriodicRefresh(sel)
+	}
+
 	return sel, nil
 }
 
+func startPeriodicRefresh(sel *AISessionSelector) {
+	clog.Infof(context.Background(), "Starting periodic refresh for Live Video to Video")
+	go func() {
+		// 5 min to avoid Ticket Params Expired and to avoid getting TTL
+		refreshInterval := 5 * time.Minute
+		ticker := time.NewTicker(refreshInterval)
+		defer ticker.Stop()
+		for {
+			select {
+			case <-ticker.C:
+				if err := sel.Refresh(context.Background()); err != nil {
```

Comment:
Might be worth a `context.WithTimeout` just in case

Reply:
Good idea, added in bf0e8671e5018f9f6abb150cc26be1da7837e22f